kafka-admin/src/main/java/io/strimzi/admin/kafka/admin/handlers/TopicsDeleteHandler.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.admin.kafka.admin.handlers;
+
+import io.strimzi.admin.kafka.admin.AdminClientProvider;
+import io.vertx.core.Promise;
+import io.vertx.ext.web.handler.graphql.VertxDataFetcher;
+import io.vertx.ext.web.impl.RoutingContextWrapper;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class TopicsDeleteHandler {
+
+    public static VertxDataFetcher deleteTopics(AdminClientProvider acp) {
+        VertxDataFetcher<List<String>> dataFetcher = new VertxDataFetcher<>((environment, prom) -> {
+            RoutingContextWrapper routingContext = environment.getContext();
+            routingContext.request().headers().get("token");
+
+            Promise deleteTopicPromise = Promise.promise();
+            List<String> names = environment.getArgument("names");
+            List<String> topicsToDelete = Arrays.asList(names.get(0).split(","));
```

Comment:
should be "names" a list of topic names that we should delete all? Why are you getting the first one and split it? It's not clear to me can you elaborate a bit more,please?

Reply:
I misunderstood the list entry in GraphQL. I used "xx, yy". It should be ["xx" "yy"].

Reply:
So a list in GraphQL is a "space" separated list and not a "comma" separated list? You should split on "space" on the list for getting all names if I got it right, or?

Reply:
Yes. However, GraphQL does the split for you. I will push fix later.